graylog2-web-interface/packages/graylog-web-plugin/package.json

```diff
@@ -49,6 +49,7 @@
     "react-dom": "16.8.6",
     "react-router": "3.2.1",
     "react-router-bootstrap": "0.23.2",
+    "react-select": "2.4.3",
```

Comment:
Why is the reason for including `react-select` here? For what I could find, so far the package is only installed in `graylog2-web-interface` and plugins use `react-select` only through the `Select` component.

Adding `react-select` here would require it to be removed from `graylog2-web-interface/package.json`, otherwise we may run into issues when the package is updated in one place and not the other.

Reply:
Thanks for the tip, I will remove it from `graylog2-web-interface/package.json`.
It needed to be added here so that the graylog plugins are able to `import` from this package.

Otherwise each plugin will need to do their own installation of the dependency.

Reply:
Sounds good to me, just be aware that we may need to update the `yarn.lock` files in the plugins as well.
Alternatively we can put it in the integrations' `package.json` for now and add it back here once more plugins depend on it.